### PR TITLE
Rename Settings action to Show ownCloud

### PR DESCRIPTION
**Pull request by TheOneRing**
Fixes: #8234

### CODE REVIEW

File: src/gui/owncloudgui.cpp [hunk 1]
```diff
@@ -814,7 +814,7 @@ void ownCloudGui::setupActions()
 {
     _actionStatus = new QAction(tr("Unknown status"), this);
     _actionStatus->setEnabled(false);
-    _actionSettings = new QAction(tr("Settings..."), this);
+    _actionSettings = new QAction(tr("Show %1").arg(Theme::instance()->appNameGUI()), this);
```
Comment (fmoc) [inline src/gui/owncloudgui.cpp:817]: I suggest to use `Show <app name> settings`, which is what we use in the desktop actions (the context menu entries on Linux desktops).

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests</strong> https://drone.owncloud.com/owncloud/client/12370/6/1

:boom: The GUI tests failed.

GUI Logs: https://cache.owncloud.com/public/owncloud/client/12370/guiReportUpload/index.html

<details>

Server Logs: https://cache.owncloud.com/public/owncloud/client/12370/guiReportUpload/serverlog.log

Screenshots:
- [Both_original_and_copied_folders_can_be_synced.png](https://cache.owncloud.com/public/owncloud/client/12370/guiReportUpload/screenshots/Both_original_and_copied_folders_can_be_synced.png)
- [File_with_long_name_(>228_characters)_is_blacklisted.png](https://cache.owncloud.com/public/owncloud/client/12370/guiReportUpload/screenshots/File_with_long_name_(>228_characters)_is_blacklisted.png)
- [File_with_long_name_can_be_synced.png](https://cache.owncloud.com/public/owncloud/client/12370/guiReportUpload/screenshots/File_with_long_name_can_be_synced.png)
- [File_with_spaces_in_the_name_can_sync.png](https://cache.owncloud.com/public/owncloud/client/12370/guiReportUpload/screenshots/File_with_spaces_in_the_name_can_sync.png)
- [Filenames_that_are_rejected_by_the_server_are_reported.png](https://cache.owncloud.com/public/owncloud/client/12370/guiReportUpload/screenshots/Filenames_that_are_rejected_by_the_server_are_reported.png)
- [Invalid_system_names_are_synced_in_linux.png](https://cache.owncloud.com/public/owncloud/client/12370/guiReportUpload/screenshots/Invalid_system_names_are_synced_in_linux.png)
- [Many_subfolders_can_be_synced.png](https://cache.owncloud.com/public/owncloud/client/12370/guiReportUpload/screenshots/Many_subfolders_can_be_synced.png)
- [Sync_all_is_selected_by_default.png](https://cache.owncloud.com/public/owncloud/client/12370/guiReportUpload/screenshots/Sync_all_is_selected_by_default.png)
- [Syncing_a_folder_to_the_server.png](https://cache.owncloud.com/public/owncloud/client/12370/guiReportUpload/screenshots/Syncing_a_folder_to_the_server.png)
- [Syncing_file_of_1_GB_size.png](https://cache.owncloud.com/public/owncloud/client/12370/guiReportUpload/screenshots/Syncing_file_of_1_GB_size.png)
- [Syncing_folders_each_having_500_files.png](https://cache.owncloud.com/public/owncloud/client/12370/guiReportUpload/screenshots/Syncing_folders_each_having_500_files.png)
- [Verify_one_empty_folder_with_a_length_longer_than_the_allowed_limit_will_not_be_synced.png](https://cache.owncloud.com/public/owncloud/client/12370/guiReportUpload/screenshots/Verify_one_empty_folder_with_a_length_longer_than_the_allowed_limit_will_not_be_synced.png)
- [Verify_pre_existing_folders_in_local_(Desktop_client)_are_copied_over_to_the_server.png](https://cache.owncloud.com/public/owncloud/client/12370/guiReportUpload/screenshots/Verify_pre_existing_folders_in_local_(Desktop_client)_are_copied_over_to_the_server.png)
- [Verify_that_you_can_create_a_subfolder_with_long_name.png](https://cache.owncloud.com/public/owncloud/client/12370/guiReportUpload/screenshots/Verify_that_you_can_create_a_subfolder_with_long_name.png)
- [change_collaborator_permissions_of_a_file_&_folder.png](https://cache.owncloud.com/public/owncloud/client/12370/guiReportUpload/screenshots/change_collaborator_permissions_of_a_file_&_folder.png)
- [sharing_a_folder_by_public_link_with_"Uploader"_role_and_check_if_file_can_be_downloaded.png](https://cache.owncloud.com/public/owncloud/client/12370/guiReportUpload/screenshots/sharing_a_folder_by_public_link_with_"Uploader"_role_and_check_if_file_can_be_downloaded.png)
- [simple_sharing_of_folder_by_public_link_with_different_roles.png](https://cache.owncloud.com/public/owncloud/client/12370/guiReportUpload/screenshots/simple_sharing_of_folder_by_public_link_with_different_roles.png)
- [various_types_of_files_can_be_synced_from_client_to_server.png](https://cache.owncloud.com/public/owncloud/client/12370/guiReportUpload/screenshots/various_types_of_files_can_be_synced_from_client_to_server.png)
- [various_types_of_files_can_be_synced_from_server_to_client.png](https://cache.owncloud.com/public/owncloud/client/12370/guiReportUpload/screenshots/various_types_of_files_can_be_synced_from_server_to_client.png)

</details>

File: src/gui/owncloudgui.cpp (hunk 1 above)
Comment (TheOneRing) [reply src/gui/owncloudgui.cpp:817]: Settings if completely missleading because we display our ui and not the settings

Comment (fmoc) [reply src/gui/owncloudgui.cpp:817]: You're right. We should possibly change this elsewhere, too. The CLI parameter `-s` stands for settings too, same goes for various context menus, etc. Perhaps we should open an issue for this.

**Review by fmoc — APPROVED** [review]

**Pull request merged**